std/array.d

```diff
@@ -451,14 +451,16 @@ if (isInputRange!Range)
 /**
 Construct a range iterating over an associative array by key/value tuples.
 
-Params: aa = The associative array to iterate over.
+Params:
+    aa = The associative array to iterate over.
 
 Returns: A $(REF_ALTTEXT forward range, isForwardRange, std,_range,primitives)
 of Tuple's of key and value pairs from the given associative array. The members
 of each pair can be accessed by name (`.key` and `.value`). or by integer
 index (0 and 1 respectively).
 */
-auto byPair(AA : Value[Key], Value, Key)(AA aa)
+auto byPair(AA)(AA aa)
```

Comment:
The previous function signature required `Value, Key` to be in the params section, so I simplified it

Reply:
Actually -- it doesn't matter. Strike the previous message.